client/custom.d.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+declare module '*.png';
```

Comment:
Any reason why this isnt in the images folder (to keep with the 'code' it represents)?